docs/Calendar/Get-CalendarDiagnosticObjectsSummary.md

```diff
@@ -48,3 +48,7 @@ Get-CalendarDiagnosticObjectsSummary.ps1 -Identity $Users -MeetingID $MeetingID
 ```
 Will create file like  `.\123456_CalLogSummary_<MeetingID>.xlsx` in current directory.
 
+
+More Documentation on collecting CalLogs and Analyzing them:
+https://learn.microsoft.com/en-us/exchange/troubleshoot/calendars/get-calendar-diagnostic-logs 
+https://learn.microsoft.com/en-us/exchange/troubleshoot/calendars/analyze-calendar-diagnostic-logs
```

Comment:
Recommend to put in aka.ms links here. 

I would also recommend to put in an aka.ms link on the main doc for the download. You can point it directly to the download, or you can point it to the docs page on mkdocs. But if you go directly to the download, then I would create another link for the mkdocs page to reference for more information. 

https://learn.microsoft.com/en-us/exchange/troubleshoot/calendars/get-calendar-diagnostic-logs#use-the-get-calendardiagnosticobjectssummaryps1-script

Reply:
Still have the old links here as well.